src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -965,9 +1178,11 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
 
     if (device === null && kind === 'video') {
       this.lastNoVideoInputDeviceCount = this.inputDeviceCount;
-      if (this.activeDevices[kind]) {
-        this.releaseMediaStream(this.activeDevices[kind].stream);
+      const active = this.activeDevices[kind];
```

Comment:
nit: Could we rename this to activeDevice? active sounds like a boolean variable :)

Reply:
Does it matter to you as a reader that you can see that the variable only exists for the next four lines?

My general rules of thumb:

* If you can see the start and end of the variable scope without moving your eye, use `a`, `d`, `c`, etc. This is like math: `F = ma` is a lot more readable in context than `acceleratingForceVector = massOfObject * accelerationVector`.
* If you can see the start and end of scope without scrolling, use just enough word to disambiguate from others: `active`, `device`, `controller`.
* If your scope is longer than that, be very explicit in one or both of name and type declaration: `activeDevice`, `CONTROLLER_TIMEOUT_MILLIS: number`, etc.